src/vs/vscode.d.ts

```diff
@@ -1989,7 +1989,7 @@ declare module 'vscode' {
 	 * { language: 'typescript', scheme: 'file' }
 	 *
 	 * @example <caption>A language filter that applies to all package.json paths</caption>
-	 * { language: 'json', scheme: 'untitled', pattern: '**​/package.json' }
+	 * { language: 'json', scheme: 'file', pattern: '**​/package.json' }
```

Comment:
I would either say "applies to all package.json paths on disk (`file` scheme)" or remove the scheme-part all together

Reply:
Gotcha, I'll remove `scheme: 'file'` since it looks redundant